packages/@guardian/source-foundations/src/colour/palette.stories.mdx

```diff
@@ -0,0 +1,162 @@
+import { Meta } from '@storybook/addon-docs';
+import { palette } from './palette';
+
+<Meta
+	title="Packages/source-foundations/palette"
+	parameters={{ previewTabs: { canvas: { hidden: true } } }}
+/>
+
+# Colour palette
```

Comment:
```suggestion
# `palette`
```

since this is the name of the export, this brings it into line with the existing examples from accessiblity

Reply:
Have changed it to 'palette' without the backticks, as it looked weird as a heading in monospace.